app/js/controllers/action_menu.js

```diff
@@ -38,8 +38,8 @@ export default class ActionMenuController extends Controller {
     this.viewSourceController.open(this.target);
   }
 
-  move() {
-    this.moveController.open(this.target);
+  copymove() {
```

Comment:
I'd recommend ```copyormove``` instead. Not a big deal, though. This change would also propagate to other places such as ```CopyOrMoveController```.